### PR TITLE
adding configuring-ptp > using-ptp redirect

### PR DESCRIPTION
**Pull request by aireilly**
Redirect for `networking/configuring-ptp.html > networking/using-ptp.html`. Merge to main only.

### CODE REVIEW

**Comment by bscott-rh** [comment]
The redirects look good to me, but we recommend creating the rules with `302` to mark temporary redirects. We migrate `302`'s to `301`'s after a month or so.

**Comment by aireilly** [comment]
> The redirects look good to me, but we recommend creating the rules with `302` to mark temporary redirects. We migrate `302`'s to `301`'s after a month or so.

Updated. Thanks.

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -579,6 +579,9 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/support/summarizing-cluster-specifications.html /dedicated/osd_support/osd-summarizing-cluster-specifications.html [NE,R=301]
     RewriteRule ^dedicated/support/?(.*)$ /dedicated/osd_support/$1 [NE,R=301]
 
+    # Redirects for PTP changes delivered in https://github.com/openshift/openshift-docs/pull/35889
+    RewriteRule ^networking/configuring-ptp.html /networking/using-ptp.html [NE,R=302]
```
Comment (gaurav-nelson) [inline .s2i/httpd-cfg/01-commercial.conf:583]: @bscott-rh Similar to my comment on the other PR, do we need `^(container-platform)/(4\.10|4\.11)` bit here as well?

Comment (aireilly) [reply .s2i/httpd-cfg/01-commercial.conf]: updated

Comment (bscott-rh) [reply .s2i/httpd-cfg/01-commercial.conf]: https://htaccess.madewithlove.com/

Comment (aireilly) [reply .s2i/httpd-cfg/01-commercial.conf]: @bscott-rh @gaurav-nelson This one can be merged now if it looks good btw. redirect should have been done months ago...

**Comment by bscott-rh** [comment]
/label merge-review-needed 

Hello merge reviewers - please merge this redirect PR to main. No cherry picks needed. Thank you

**Pull request merged**